jetty-ee10/jetty-ee10-maven-plugin/src/main/java/org/eclipse/jetty/ee10/maven/plugin/WebAppPropertyConverter.java

```diff
@@ -214,7 +215,7 @@ public static void fromProperties(MavenWebAppContext webApp, Properties webAppPr
         // - the tmp directory
         str = webAppProperties.getProperty(TMP_DIR);
         if (!StringUtil.isBlank(str))
-            webApp.setTempDirectory(new File(str.trim()));
+            webApp.setTempDirectory(IO.asFile(str));
```

Comment:
I'd prefer `Resource.asFile(str)`, here and in the other places.

Reply:
Left alone, see above.